apps/minifront/src/components/swap/swap-form/simulate-swap-result/trace/price.tsx

```diff
@@ -23,14 +23,22 @@ export const Price = ({
     if (firstValueMetadata?.symbol && lastValueMetadata?.symbol) {
       const inputDisplayDenomExponent = getDisplayDenomExponent.optional()(firstValueMetadata) ?? 0;
       const outputDisplayDenomExponent = getDisplayDenomExponent.optional()(lastValueMetadata) ?? 0;
-      const formattedInputAmount = formatAmount(inputValue.amount, inputDisplayDenomExponent);
-      const formattedOutputAmount = formatAmount(outputValue.amount, outputDisplayDenomExponent);
+      const formattedInputAmount = formatAmount({
+        amount: inputValue.amount,
+        exponent: inputDisplayDenomExponent,
+      });
+      const formattedOutputAmount = formatAmount({
+        amount: outputValue.amount,
+        exponent: outputDisplayDenomExponent,
+      });
 
       const outputToInputRatio = new BigNumber(formattedOutputAmount)
         .dividedBy(formattedInputAmount)
         .toFormat(outputDisplayDenomExponent);
 
-      price = `1 ${firstValueMetadata.symbol} = ${outputToInputRatio} ${lastValueMetadata.symbol}`;
+      const outputToInputFormatted = removeTrailingZeros(outputToInputRatio);
```

Comment:
Removing trailing zeros from the format so we don't have outputs like:

<img width="301" alt="Screenshot 2024-05-31 at 6 29 23 PM" src="https://github.com/penumbra-zone/web/assets/16624263/d4c2f736-1b93-414b-99f1-e40ae0a5627f">